guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -522,6 +528,9 @@ class View extends Template
 
 6.2.4. Actions MUST NOT reference blocks declared in layout.
 
+6.2.5. Configuration for the presentation layer MUST be declared in the corresponding application area.
```

Comment:
6.2.5 should be combined with 3.4

Reply:
This is not exactly the same, 3.4 says only about DI config.